packages/bootstrap/scss/appbar/_theme.scss

```diff
@@ -0,0 +1,7 @@
+$appbar-light-bg: #ffffff !default;
+$appbar-light-text: #6c757d !default;
+
+$appbar-dark-bg: #343a40 !default;
+$appbar-dark-text: rgba(#f8f9fa, .6) !default;
```

Comment:
That should be part of the _variables file for each theme

Reply:
Ok, I'll extract them.